bluemira/equilibria/coils/_grouping.py

```diff
@@ -932,19 +934,24 @@ def control(self) -> List:
         return self._control
 
     @control.setter
-    def control(self, control_names: Optional[Union[List, bool]] = None):
-        """Set control coils"""
+    def control(self, control_names: Optional[Union[List[str], bool]] = None):
+        """Set which coils are actively controlled
+        control names: can be one of the following:
+                    - list of str, each one being the name of each control coil.
+                    - None, for when ALL coils are control coils.
+                    - a boolean, which denotes all controlled vs none controlled.
+        """
```

Comment:
slight layout adjustment
```suggestion
        """Set which coils are actively controlled

        Parameters
        ----------
        control_names:
                    - list of str, each one being the name of each control coil.
                    - None, for when ALL coils are control coils.
                    - a boolean, which denotes all controlled vs none controlled.
        """
```